.github/workflows/examine-wasm.yml

```diff
@@ -0,0 +1,44 @@
+name: Examine Wasm compatibility
```

Comment:
I already added this, it's `wasm.yml`. Remove this file.

Reply:
illl delete the other one, i like keeping the examine in the name. our workflow folder is starting to get messy